.circleci/continue_config.yml

```
@@ -0,0 +1,249 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.4.0
+
+# the default pipeline parameters, which will be updated according to
+# the results of the path-filtering orb
+parameters:
+  run-build-kedro-telemetry:
+    type: boolean
+    default: false
+  run-build-kedro-docker:
+    type: boolean
+    default: false
+  run-build-kedro-airflow:
+    type: boolean
+    default: false
+
+
+commands:
+  setup_conda:
+    parameters:
+      python_version:
+        type: string
+    steps:
+      - run:
+          name: Create virtual env
+          command: |
+            # Get rid of pyenv stuff
+            sudo rm -rf .pyenv/ /opt/circleci/.pyenv/
+            # Download and install miniconda
+            curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh  > miniconda.sh
+            bash miniconda.sh -b -p $HOME/miniconda
+            # Create an anaconda virtualenv for python_version and make that the default python interpreter
+            echo ". /home/circleci/miniconda/etc/profile.d/conda.sh" >> $BASH_ENV
+            echo "conda deactivate; conda activate kedro_plugins" >> $BASH_ENV
+            . /home/circleci/miniconda/etc/profile.d/conda.sh
+            conda create --name kedro_plugins python=<<parameters.python_version>> -y
+            source $BASH_ENV
```

```suggestion
            echo ". /home/circleci/miniconda/etc/profile.d/conda.sh" >> $BASH_ENV
            echo "conda deactivate; conda activate kedro_plugins" >> $BASH_ENV
            . /home/circleci/miniconda/etc/profile.d/conda.sh
            conda create --name kedro_plugins python=<<parameters.python_version>> -y
            source $BASH_ENV
```

This one seems mixed up. Why is `$BASH_ENV` overwritten before being used at all? And then why do we not just call `conda deactivate; conda activate kedro_plugins` directly instead of echoing it to a variable?

---

I haven't written the whole setup from scratch, so in all honesty I've been a bit lazy and left some things as they were and focussed more and getting the builds to run properly. If it's okay with you I'd like to get this merged and will then focus on clean up, because it's already quite a lot to review.

---

I have split this chunk in separate run steps so hopefully it's more clear now what's happening.